orchagent/port.h

```diff
@@ -8,11 +8,28 @@ extern "C" {
 #include <set>
 #include <string>
 #include <vector>
+#include <map>
 
 #define DEFAULT_PORT_VLAN_ID    1
 
 namespace swss {
 
+struct VlanMemberEntry
+{
+    sai_object_id_t            vlan_member_id;
+    sai_vlan_tagging_mode_t    vlan_mode;
+};
+
+typedef std::map<sai_vlan_id_t, VlanMemberEntry> port_vlan_members_t;
+
+struct VlanInfo
+{
+    sai_object_id_t     vlan_oid;
+    sai_vlan_id_t       vlan_id;
+    std::string         autostate;
+    uint32_t            mtu;
```

Comment:
move this mtu to Port structure's variable

Reply:
ok.